frontend/src/actions/parcelsActions.ts

```diff
@@ -70,7 +70,9 @@ export enum LeasedLand {
 
 export interface IBuilding extends IProperty {
   parcelId: number;
+  pid: number;
   address: IAddress;
+  administrativeArea: string;
```

Comment:
Currently this is what's being returned by the map (address is just a string rather than an IAddress), is this what your PR is changing? 
![image](https://user-images.githubusercontent.com/33818575/106177241-1b1a0c00-614d-11eb-96b2-1857ec60cf3c.png)

Reply:
The map uses GeoJSON which is a flat model of values for the property.  However when displaying the information in the sidebar or on Inventory List View it should be in a more standard format `IParcel` or `IBuilding`.  These have an `IAddress` property `property.address.administrativeArea`.  My PR has a converter that ensures the GeoJSON is changed to match the correct `IParcel` or `IBuilding` interface.  It calls this converter before sending the data to the Popup.

Reply:
I've copied over your converter, thank you!